topic-operator/src/main/java/io/strimzi/operator/topic/TopicOperator.java

```diff
@@ -1202,9 +1203,10 @@ public ReconcileState(Set<TopicName> succeeded, Set<TopicName> undetermined, Map
             this.succeeded = succeeded;
             this.undetermined = undetermined;
             this.failed = failed;
+            this.ktList = emptyList();
```

Comment:
Again, I don't understand why this needed to change. Presumably because you saw an NPE, right? But the question is not "How do I get rid of the NPE as quickly as possible" it's "what race means we're not seeing this NPE, and was that an NPE the original author thought could never happen?" The answer to that question might reveal some defect in the logic which using an `emptyList()` is just hiding.

Reply:
I did not see the NPE. Spotbugs with java 11 started complaining about dereferencing `ktList` which has not been initialized in constructor.